docs/python.org/code-of-conduct/Procedures-for-Reporting-Incidents.md

```diff
@@ -14,18 +14,15 @@ The best way to contact the PSF Code of Conduct working group is by email at **<
 
 In the event of a [conflict of interest](#conflicts-of-interest), you may directly contact any of the lead incident responders:
 
-  * Deb Nicholson
-      * Python Software Foundation - Executive Director
-      * <deb@python.org>
-  * Łukasz Langa
-      * Python Software Foundation - Code of Conduct WG Chair
-      * <lukasz@python.org>
-  * Tania Allard
-      * Python Software Foundation Director
-      * <tania@python.org>
-  * Drew Winstel
-      * Python Code of Conduct WG Member
-      * <drew@defna.org>
+* Deb Nicholson
```

Comment:
@trallard can you re-structure the markdown lists as they were before to restore the nested structure?

Before:

<img width="600" alt="Screenshot 2025-02-13 at 12 10 29 PM" src="https://github.com/user-attachments/assets/aeeab928-c38c-4186-a0ff-a4f1122d7f48" />

After:

<img width="600" alt="Screenshot 2025-02-13 at 12 09 16 PM" src="https://github.com/user-attachments/assets/66550183-88e3-4275-a14d-818aaa003a2e" />

Reply:
Done! Did not realise I had messed it up, sorry